src/main/java/longtimenosee/model/person/predicate/EmailMatchesInputPredicate.java

```diff
@@ -23,7 +23,7 @@ public EmailMatchesInputPredicate(String input) {
 
     @Override
     public boolean test(Person person) {
-        return person.getEmail().value.equals(input);
+        return person.getEmail().value.equalsIgnoreCase(input);
```

Comment:
Not too sure on this, but maybe state what `equalsIgnoreCase` does in a comment?
Alternatively, maybe can add a javadoc comment specifying that the predicate test is not case sensitive!